pkg/ruler/ruler.go

```diff
@@ -1087,7 +1095,7 @@ func (r *Ruler) getLocalRules(userID string, req RulesRequest) ([]*GroupStateDes
 				if !getAlertingRules {
 					continue
 				}
-				rule.ActiveAlerts()
```

Comment:
Note to reviewers: this function just return active alerts. There's no point calling it. It called again a couple of lines below, where we iterate on active alerts. I believe this function call is a typo.